pynvjitlink/tests/test_pynvjitlink.py

```diff
@@ -98,16 +98,16 @@ def test_complete_empty():
         ("device_functions_archive", InputType.LIBRARY),
     ],
 )
-def test_add_file(input_file, input_type, request):
```

Comment:
I'll comment on the nearest line to L93, for ltoir test case, wouldn't you just need to add `-lto` flag to `.create` method?

Reply:
Thanks for pointing this out... The input type was also wrong - it's not "raw" LTO-IR, it's wrapped in an ELF, so the input type needed to be `OBJECT`, which nvJitLink then extracts the LTO-IR from. Because it's a little different to the other cases, I've moved it into a separate test to fix up.